galasa-parent/build.gradle

```diff
@@ -35,7 +35,7 @@ allprojects {
         mavenCentral()
         mavenLocal()
```

Comment:
Shouldn't the mavenlocal be first ? and the mavenCentral be last ? I think it might make a difference in the order of preference where gradle gets things from...

Reply:
I didn't know that thanks for pointing that out. I agree that order makes more sense, so I'll update it.